README.md

```diff
@@ -11,6 +11,12 @@ The Beacon SDK 2.0 requires iOS 11.0+.
 As of Beacon 2.1.0 Xcode 12 is required to build with the Beacon SDK.
 Beacon 2.0.x requires Xcode 11.4
 
+### Supported platform and language versions
+
+* iOS 11.0 to 14.0
+* Swift 5.4
```

Comment:
```suggestion
* Swift 5
```

Reply:
Sorry @ScottPetit this and the other points were totally a cut and paste issue! I was listening to you when you mentioned to drop the minor version 🤦

Reply:
Haha all good, figured as much.